crates/containerd-shim-wasm/Cargo.lock

```diff
@@ -1,1476 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
Why is this lock file and the other one below needless?
These are just generated by cargo.

Reply:
I think maybe those files were generated outside from this repo. In my test, those lock files are not generated and update automatic.

[Ref](https://doc.rust-lang.org/beta/book/ch14-03-cargo-workspaces.html)
Notice that the workspace has only one Cargo.lock file at the top level, rather than having a Cargo.lock in each crate’s directory.